resources/kids.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+      "name": "Pixie Fizz",
+      "dob": "01.02.2012",
+      "tooth": "22.07.2012",
```

Comment:
`birthDate` over `dob` ^^

I would also use `firstToothDate`.

Also, use ISO Date format, like `2012-02-01` and `2012-07-22`, then you can format it inside your components.

Reply:
Thanks! I will do that.